hasher-matcher-actioner/webapp/src/pages/settings/ActionSettingsTab.jsx

```diff
@@ -123,7 +123,14 @@ export default function ActionSettingsTab() {
       <Card>
         <Card.Header>
           <h2 className="mt-2">Action Definitions</h2>
-          <h5 className="mt-5">Define what to do for different Actions</h5>
+          <h5 className="mt-5">
```

Comment:
The length of this text is a bit long for a heading (even an h5). Maybe consider `<p>` instead?

Reply:
Also consider shortening the text. There wasn't an example given for the ActionRules description, for example.

Reply:
I think its better to be verbose at this stage of the project. Too much room for confusion. Will convert to `<p>` though